magicctapipe/scripts/lst1_magic/semi_automatic_scripts/clusters.py

```diff
@@ -35,6 +40,7 @@ def slurm_lines(queue, job_name, array=None, mem=None, out_name=None):
         "#SBATCH -n 1\n\n",
         f"#SBATCH --output={out_name}.out\n" if out_name is not None else "",
         f"#SBATCH --error={out_name}.err\n\n" if out_name is not None else "",
+        f"#SBATCH --nice={nice_parameter}\n",
```

Comment:
please add ```if out_name is not None else "",``` as in the lines above, otherwise it will try to set --nice=None that will not work as Elisa pointed out